.github/workflows/luarocks.yml

```diff
@@ -0,0 +1,42 @@
+name: Luarocks
+
+on:
+  push:
+    tags:
+      - '*'
+  release:
+    types:
+      - created
+  pull_request: # Tests packaging on PR
+  workflow_dispatch:
+
+jobs:
+  luarocks-upload:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Required to get the tags
+      - name: Get Version
+        run: echo "LUAROCKS_VERSION=$(git describe --abbrev=0 --tags)" >> $GITHUB_ENV
+
+      - name: Install C/C++ Compiler
+        uses: rlalik/setup-cpp-compiler@master
+        with:
+          compiler: clang-latest
+
+      - name: Install tree-sitter CLI
+        uses: baptiste0928/cargo-install@v3
+        with:
+          crate: tree-sitter-cli
+
+      - name: LuaRocks Upload
+        uses: nvim-neorocks/luarocks-tag-release@v5
```

Comment:
Neat action :)

If we use `@v7` do the labels get picked up, or is this a more general problem?

https://github.com/nvim-neorocks/luarocks-tag-release?tab=readme-ov-file#labels

Reply:
Oops, my bad. This should be using v7.
v5 tries to run busted tests if a .busted file is present. We moved that to a separate action, because it caused issues with this one.

The labels problem is a luarocks-site bug.